metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -392,6 +400,37 @@ def get_aspects_for_entity(
 
         return result
 
+    def get_entity_semityped(self, entity_urn: str) -> AspectBag:
```

Comment:
Thoughts on the name `get_entity_aspects_semityped`? The doc explains the name but alone it doesn't make sense to me. Or honestly think you could do `get_entity_aspects_typed` and that'd be fair

Reply:
yep - my reasoning is that eventually I want to add a plain `get_entity` method that returns a first-class "Entity" that is actually strongly typed and has the exact set of aspects supported by that entity (rather than this semityped one, which can return any aspect that we know about)

Given that these methods/classes don't have super high usage, imo it's fine. We can also rename to `get_entity_aspects_typed ` or `get_entity_aspect_bag` or something similar in the future.